src/utils/viewer/viewer.cpp

```diff
@@ -570,10 +610,47 @@ void Viewer::viewportEvent(ViewportEvent& event) {
   renderCamera_->setViewport(event.windowSize());
   imgui_.relayout(Mn::Vector2{event.windowSize()} / event.dpiScaling(),
                   event.windowSize(), event.framebufferSize());
+
+  objectPickingHelper_->handleViewportChange(event.framebufferSize());
+}
+
+void Viewer::createPickedObjectVisualizer(unsigned int objectId) {
+  for (auto& it : sceneGraph_->getDrawableGroups()) {
+    if (it.second.hasDrawable(objectId)) {
+      auto* pickedDrawable = it.second.getDrawable(objectId);
+      objectPickingHelper_->createPickedObjectVisualizer(pickedDrawable);
+      break;
+    }
+  }
 }
 
 void Viewer::mousePressEvent(MouseEvent& event) {
+  if (event.button() == MouseEvent::Button::Right &&
+      (event.modifiers() & MouseEvent::Modifier::Shift)) {
```

Comment:
Is there some existing documentation for the viewer utility (such as [magnum-player](https://doc.magnum.graphics/magnum/magnum-player.html#magnum-player-controls) has) where this new key control could get listed?

Otherwise <kbd>Shift</kbd> + <kbd>right click</kbd> is quite hard to discover I fear.

Reply:
Very good point. In the previous version, I put it in the keyPressEvent function. So, somehow people can know this functionality.
Then I simplified the code in this way as you suggested.

No, we do not have one to describe the functionality. Usually we go and read the keyPressEvent().